host/xtest/Makefile

```diff
@@ -10,6 +10,10 @@ endif
 # be used instead.
 OPTEE_CLIENT_EXPORT ?= ../../../optee_client/out/export
 
+# If CFG_XTEST_LITE=n : compile full xtest suite (default)
+# If CFG_XTEST_LITE=y : compile sanity xtest suite
+CFG_XTEST_LITE ?= n
```

Comment:
It can be a simple `if` I think: what about adding  global variable (`option_light` for instance) that would be set when the command line is parsed, and could be tested from anywhere? Personally I don't like the idea of adding things when we know we want to remove them ASAP.